### ISSUE TITLE
Assignments, questions, reminders

### ISSUE THREAD

**Issue by rainersachs** [rainersachs/mouse.HG.2019.plus#1]

**Comment by rainersachs** [comment]
Henry: your assignments for the first part of the semester follow. Take your time and do a careful job. There is no deadline.

1. Simplify and then freeze our brenner/shuryak repository. Try to fix it so that if a newbie or I unfreeze it and push the switch it will run and we will be able to see how to improve it. Put in your constant dose rate code with good DERs whose parameters have already been assigned values or distributions and contain more than 1 and less than 5 or 6  pre-adjusted adjustable parameters. Tell us what to do. Make it a default procedure where we have no options or very few options. By commenting the code and/or notes within this issue indicate briefly why you used any particular chunk. When you are done and ready to freeze I will test it and perhaps suggest ways to make it more nearly self explanatory. Then freeze it and remove yourself from the collaborators.

2. I will email you versions of the paper and its web supplement that I just finished today. Please start reading them critically (paper first then the web supplement) keeping track of all your improvements and questions). This is a excellent fit. The paper is waiting for Ellie and Polly to put in the experimental part but the theory should be and I think is already nearly polished and self consistent. You are by now well equipped to understand  everything in the paper if you read carefully and check stuff with the code (which I will upload to this repository by Monday). But you are not so familiar with the material that you can read the paper without using a microscope so you will not fall into whatever traps I have fallen into, you will catch my mistakes better than Edward or Peter could because they already know too much. So you will also be learning new material that you need to know when going forward.

3. Take charge. Decide when you and I should meet, when we should talk on the phone, when we should email, when we should put stuff in this repository. Assume we will have 1 newbie and suggest what assignments the newbie should be given. At least at first funnel the newbies assignments through me. Try to anticipate what tweaks will be needed in later more important papers by finding out from Ellie and Polly what new mixed field HG data is in the 16 pipeline or scheduled to be carried out at NSRL. etc.

To repeat, take your time, don't spend inordinate amounts of time per week, don't rush. I think you can do all that in one semester.  That would already be very useful to me, to you, to Ellie and Polly and with any luck to the newbie. Maybe, because I sometimes underestimate how fast you calculate and read, you will only need half a semester or even less.

**Comment by rainersachs** [comment]
The plan is to have this repository as the master for the LSSR paper. For at least the first part of Fall 2019 semester Henry, Edward, and Ray should be the only collaborators that can push. Newbies Grace and Liyang should get their information from this repository. and participate in the discussions in this issue#1, in boxes such as this one, but push only by sending stuff to me or Henry and letting us push for them'

**Comment by rainersachs** [comment]
[existence.pdf](https://github.com/rainersachs/Chang2019_V1/files/3588193/existence.pdf)
to gracie and liyang: a long explanation of your assignments for the first part of the semester will follow in this issue soon. Here are some relevant .pdf files
https://en.wikipedia.org/wiki/Picard%E2%80%93Lindel%C3%B6f_theorem

**Comment by rainersachs** [comment]
Hi Henry, Gracie, and Liyang!
All 3 meetings with Sachs are in his office this week: Gracie at around 11, Liyang at 12:15, Henry at 1:45.  I hope Gracie and Henry can arrange for a 10 minute period when all 3 of us meet together.
1.	This GitHub  entry concerns a newly revised version of the class paper and starting to "presubmit" it to the LSSR (Life Sciences in Space Research) journal.
2.	Presubmission will take about a month – it is unfortunately clumsy. It is done via the journal's web site. 
3.	Liyang: you and I will work on fine tuning the science in the class paper as Gracie and Henry work on presubmitting it – the presubmission process entails their frequently revising the files until near the end of the process. Please focus all further efforts until the presubmission process is finished on revising the class paper, using the GitHub code in some cases. That is much the best way for you to learn what you need to know for the class, learn how the code works, and help increase our chances of acceptance by the journal.
4.	Liyang and Gracie: In this box are about 5 .pdf files of published LSSR papers on random space science topics. When revising our paper it will often be useful to see how other authors handle various items like numbering, figure captions, table captions, etc.
5.	Henry and Gracie. Your job will teach you stuff useful later when you start to publish papers and will teach you some science. Unfortunately it will also involve some confusing items which are basically just filling out a complicated interrelated set of website forms (see item 2 above!). Start by reading the "instruction to authors" .pdf for LSSR (and for the publisher if the publisher has a different  instructions for authors .pdf). Then see if you can sign up for and get into the submissions website. If you get in make a mock paper (title box: "HG by Henry"; abstract box: "Mousies and Gracie"; etc.). See what further instructions the site gives you and see what happens when you try to follow those instructions. After a while, if the site lets you, delete the entire mock paper ASAP.
6.	Gracie: When we actually presubmit I will have to be the corresponding author, Try to find out if LSSR allows me to deputize you to fill in the website so that all I have to do is check everything at the end and push the submit button (and, in practice, almost always get an error message which you, Henry or I will have to try to fix)
7.	Gracie your minutes were good. Please put them with an informative title in a box of this "issues"
8.	Henry: Have you finished putting your Shuryak and Brenner code in mothballs? If so tell me the name of the repository so I can make it private. 
[1-s2.0-S2214552417301281-main.pdf](https://github.com/rainersachs/mouse.HG.2019.plus/files/3614393/1-s2.0-S2214552417301281-main.pdf)
[1-s2.0-S2214552418300671-main.pdf](https://github.com/rainersachs/mouse.HG.2019.plus/files/3614394/1-s2.0-S2214552418300671-main.pdf)
[authorPack.pdf](https://github.com/rainersachs/mouse.HG.2019.plus/files/3614395/authorPack.pdf)
[failure_mode_nano.pdf](https://github.com/rainersachs/mouse.HG.2019.plus/files/3614396/failure_mode_nano.pdf)
[late_hippo_H1_effects.pdf](https://github.com/rainersachs/mouse.HG.2019.plus/files/3614397/late_hippo_H1_effects.pdf)
[LSSR_author_info.pdf](https://github.com/rainersachs/mouse.HG.2019.plus/files/3614398/LSSR_author_info.pdf)

**Comment by RenYiWang** [comment]
Yes, I have finished reviewing Shuryak and Brenner in the repository "Shuryak_Brenner." However, I do not know how to freeze a repository. The link is here: https://github.com/rainersachs/Shuryak_Brenner

**Comment by rainersachs** [comment]
I just got wind of a paper whose senior (i.e. last) author is Martin Kuiper, Norwegian University of Science and Technology, Trondheim (NTNU). Googling shows that Kuiper must be a well respected statistics-oriented modeler who knows a lot about synergy among drugs that are used to combat cancer, including colorectal cancer (which is a modelers' favorite because an enormous amount is known about it at a molecular-omics level and also at much larger scales).Googling makes me think perhaps Kuiper's other papers may have something that I have long been looking for: high precision data on therapeutic drug synergies that can be used to compare IEA with SEA in a paper for modern main-line biostatisticians. So far a lot of my efforts to find such data have failed because the DERs used are too crude. 
A project useful for a URAP student is the following. Find a few promising recent papers by Kuiper and coworkers by skimming abstracts in PubMed or "my NCBI" or Medline, the canonical biology literature data base(s). Study those papers in detail, and decide if IEA synergy theory can be used on the data the papers make available or the authors might be willing to make available, perhaps while asking for a joint paper. The student would learn a lot about the following. IEA; cancer as studied by modern main-line biostatisticians; cancer at the clinical level; branches of advanced statistics not mentioned in undergraduate courses; the situation at the very start of any research – exploratory literature searches or inquiries to experts (e.g. UCB professors in the student's statistics classes) to find out if the possible research project is promising (the initial floundering around unsystematically stage of all research!); a possible future research area for the student's own career; the dancing around that occurs when trying to establish collaborations with other teams; etc.
     I am quite excited about this project, but have learned that most of the time such hot early ideas fizzle out for various reasons. In research one just needs to keep producing such ideas and with luck one of them sooner or later leads to solid results.

**Comment by rainersachs** [comment]
Hi Gracie!

We will almost certainly submit to LSSR rather than another journal. During our meeting tomorrow my agenda is to discuss how I would like you to help me with the submission process next week. Here is how that would start. Log in to the LSSR author submission Website, as my assistant if they will allow that and as yourself if they do not. The title and abstract of the attached are in final form (though the rest of the MS will need some revisions in the details) and the first thing you will do is copy title and abstract, then fill in their boxes on the website. That activates the submission and gives the manuscript a label that it will carry throughout the submission processTake the attached and put in your computer, in a directory/folder dedicated to the submission. See if you can make endnote turn the curly brackets in the attached into actual inline citations and simultaneously add the bibliography, which is missing in the attached. If you can go back and forth between curly brackets and bibliography we are all set as far as endnote goes, tweaks apart. If not, we need to see how handle endnote, preferably by having URAP buy you a good version. Once that is done you should clone the attached, rename the attached by adding "frozen version" to the name and freeze the frozen version, by insisting it always opens as read only and never using allow editing. Then you can start dismantling the unfrozen version (use the curly bracket form until near the very end). The website will tell you what dismantled pieces it wants. Practise that a little if you have time but only a little, and then pause for further explanations from me.
--
